deployment/helm/human-connection/templates/deployments/deployment-neo4j.yaml

```diff
@@ -33,15 +38,6 @@ spec:
             memory: {{ .Values.neo4jResourceRequestsMemory | default "1G" | quote }}
           limits:
             memory: {{ .Values.neo4jResourceLimitsMemory | default "1G" | quote }}
-        env:
-        - name: NEO4J_dbms_security_procedures_unrestricted
-          value: "algo.*,apoc.*"
```

Comment:
Obsolete? We don't need that anymore?

Reply:
we do need it actually! I ran into an issue with running the `yarn prod:migrate init` yesterday on the `develop` server because this was removed... I'll add it back.

I did have a question about whether we should have `production` config stored locally and `.gitignored`, or how we should handle that? I would like to have a copy of what is being deployed in production in case there is a need to recreate the cluster for some reason.

I have looked into `helm-secrets` and got it installed on my computer, but when I tried to use it, it threw errors. Before debugging, I though it would be best to come to some consensus on it.

Reply:
Actually, I hadn't realized that you are `.gitignore`ing the `configmap.yaml` and `secrets.yaml` ... I have extracted all the `env` variables there.
My suggestion would be for myself and you to have different files like `develop.configmap.yaml`/`production.configmap.yaml` or different `values.yaml` and `.gitignore` them.